eeps/eep-0072.md

```diff
@@ -0,0 +1,195 @@
+    Author: Roberto Aloi <prof3ta(at)gmail(dot)com>
+    Status: Draft
+    Type: Standards Track
+    Created: 23-Aug-2024
+    Erlang-Version: OTP-28
+    Post-History:
+****
+EEP 72: Erlang Error Index
+----
+
+Abstract
+========
+
+The **Erlang Error Index** is a _catalogue_ of errors emitted by
+various tools within the Erlang ecosystem, including - but not limited
+to - the `erlc` Erlang compiler and the `dialyzer` type checker.
+
+The catalogue is not limited to tools shipped with Erlang/OTP, but it
+can include third-party applications such as the [EqWAlizer][]
+type-checker or the [Elvis][] code style reviewer.
+
+Each error in the catalogue is identified by a **unique code** and it
```

Comment:
Should codes have a human-readable alias? How would we incorporate the "namespace" into the alias?

Reply:
I like standardized human-readable aliases; they may not need to be unique (e.g. parsing errors, eqwalizer/dialyzer may report the same thing with a different error)